core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/request/resource/BaseGetResourceRequestBuilder.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 B2i Healthcare Pte Ltd, http://b2i.sg
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.b2international.snowowl.core.request.resource;
+
+import com.b2international.snowowl.core.context.ResourceRepositoryRequestBuilder;
+import com.b2international.snowowl.core.domain.RepositoryContext;
+import com.b2international.snowowl.core.request.GetResourceRequestBuilder;
+import com.b2international.snowowl.core.request.SearchResourceRequestBuilder;
+
+/**
+ * @since 8.7
+ */
+public abstract class BaseGetResourceRequestBuilder<
+	B extends GetResourceRequestBuilder<B, SB, RepositoryContext, SR, R>, 
+	SB extends SearchResourceRequestBuilder<SB, RepositoryContext, SR>, 
+	SR, 
+	R> extends GetResourceRequestBuilder<B, SB, RepositoryContext, SR, R> implements ResourceRepositoryRequestBuilder<R> {
```

Comment:
😖 Please move this last `extends` to its own line, so there is a separation between the type parameters and the base class of this builder.